src/components/Form/SetAclPermsDocContainerForm.jsx

```diff
@@ -84,22 +89,31 @@ const SetAclPermsDocContainerForm = () => {
       return;
     }
 
-    try {
-      await setDocContainerAclPermission(session, permissions, podUrl, otherPodUsername);
+    // Routine for setting permissions to Documents
+    if (user === 'personal') {
+      try {
+        await setDocContainerAclPermission(session, permissions, podUrl, otherPodUsername);
 
-      runNotification(
-        `${permissions.read ? 'Give' : 'Revoke'} permission to ${
-          selectedUser.person ?? username
-        } for Documents Container.`,
-        5,
-        state,
-        dispatch
-      );
-      setTimeout(() => {
-        clearInputFields();
-      }, 3000);
-    } catch (error) {
-      runNotification('Set permissions failed. Reason: File not found.', 5, state, dispatch);
+        runNotification(
+          `${permissions.read ? 'Give' : 'Revoke'} permission to ${
+            selectedUser.person ?? username
+          } for Documents Container.`,
+          5,
+          state,
+          dispatch
+        );
+        setTimeout(() => {
+          clearInputFields();
+        }, 3000);
+      } catch (error) {
+        runNotification('Set permissions failed. Reason: File not found.', 5, state, dispatch);
+        setTimeout(() => {
+          clearInputFields();
+        }, 3000);
+      }
+    } else {
+      // Routine for requesting permissions to Documents
```

Comment:
I don't think we've finalized with UI/UX what we want permission requests to look like. It will probably look different than this. I think these TODO comments confuse more than they help.

Reply:
We'll eventually need a request permissions functions. We don't have to act on them now (we can save it for another PR), but could be a good reminder.